ledger-tool/src/main.rs

```diff
@@ -1222,13 +1222,18 @@ fn load_bank_forks(
 
         let (confirmed_bank_sender, confirmed_bank_receiver) = unbounded();
         drop(confirmed_bank_sender);
-        let geyser_service =
-            GeyserPluginService::new(confirmed_bank_receiver, &geyser_config_files).unwrap_or_else(
-                |err| {
-                    eprintln!("Failed to setup Geyser service: {err:?}");
-                    exit(1);
-                },
-            );
+
+        // solana-ledger-tool doesn't run an admin_rpc service, so just pass in a dummy rx
+        let rpc_to_plugin_manager_rx = None;
```

Comment:
I think Tyera made a point before, name it to rpc_to_plugin_manager_receiver -- it is easier to read. So please avoid rx, tx acronyms here.

Reply:
Hmm I thought I got them all. Missed this one